### PR TITLE
Remove needless lifetimes

### PR DESCRIPTION
**Pull request by jeremystucki**
changelog: none

### CODE REVIEW

**Comment by flip1995** [comment]
Thanks!

@bors r+

File: clippy_lints/src/missing_inline.rs [hunk 1]
```diff
@@ -68,7 +68,7 @@ fn check_missing_inline_attrs(cx: &LateContext<'_, '_>, attrs: &[ast::Attribute]
     }
 }
 
-fn is_executable<'a, 'tcx>(cx: &LateContext<'a, 'tcx>) -> bool {
+fn is_executable<'a>(cx: &LateContext<'a, '_>) -> bool {
```
Comment (lzutao) [inline clippy_lints/src/missing_inline.rs]: Could we remove `'a` lifetime?

Comment (flip1995) [reply clippy_lints/src/missing_inline.rs]: Good catch!

**Pull request merged**